pwiz_tools/Skyline/Skyline.csproj

```diff
@@ -343,7 +355,9 @@
     <Compile Include="Controls\AuditLog\AuditLogForm.Designer.cs">
       <DependentUpon>AuditLogForm.cs</DependentUpon>
     </Compile>
-    <Compile Include="Model\AuditLog\AuditLogEntry.cs" />
+    <Compile Include="Model\AuditLog\AuditLogEntry.cs">
+      <SubType>Form</SubType>

```

Comment:
Yes, at some point I found myself inheriting the AuditLogFormSettings class from a class that does not even belong to a form, so renaming it would be a good idea. I already changed the AuditLogForm to be an interface yesterday.